parlai/zoo/multimodal_blender/README.md

```diff
@@ -0,0 +1,145 @@
+# Model card: Multi-Modal BlenderBot
+
+Model card for the MMB DegenPos model described in [Multi-Modal Open-Domain Dialogue](TODO: add arXiv link).
+
+## Model details
+This model was trained to create a dialogue agent that can converse engangly about the content of an image as well as about general chitchat. It is a 2.7-billion-parameter Transformer sequence-to-sequence model based on the [BlenderBot](https://ai.facebook.com/blog/state-of-the-art-open-source-chatbot/) open-domain chatbot, trained with image features from [COCO Captions](https://cocodataset.org/) and [Image-Chat](https://parl.ai/projects/image_chat/), and using an image encoder from [Faster R-CNN](https://arxiv.org/abs/1506.01497?context=cs).
```

Comment:
nit: "engagingly"

also, "trained with image features from X datasets" sounds a little off, as the model was trained on those tasks not necessarily just their image features - perhaps "trained on image tasks - [COCO Captions] and [Image-Chat] - using an image encoder from ..."

Reply:
yes - changed